org.rcpml.core/src/org/rcpml/core/RCPML.java

```diff
@@ -33,4 +36,27 @@ public static IController create(Document document) {
 	public static IController createWithConstructor(Document document) {
 		return new Controller(document, true);
 	}
+	
+	public static Composite createComposite(Document doc, Composite parent) {
+		Object presentation = RCPML.renderDocument(doc);
+		Composite composite = getComposite(presentation, parent);
+		return composite;
+	}
+	
+	private static Composite getComposite(Object presentation, Composite parent) {
```

Comment:
[minor] 
This getter method has a side-effect (changing parent composite)
Move parent  composite.setParent(parent); into caller method.